### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by Akhil-eastern**

### CODE REVIEW

File: src/pages/PageTransferAttend/index.tsx [hunk 1]
```diff
@@ -19,7 +20,7 @@ import CustomFonts from 'utils/CustomFonts'
 @observer
 class PageTransferAttend extends React.Component {
   render() {
-    const currentCall = callStore.currentCall
+    const currentCall: any = callStore.currentCall || {}
```
Comment (jigarshah1902) [inline src/pages/PageTransferAttend/index.tsx:23]: Destructure callstore, and do not use type any

File: src/pages/PageTransferAttend/index.tsx [hunk 2]
```diff
@@ -43,7 +44,7 @@ class PageTransferAttend extends React.Component {
       transferingAvatarName = transferingAvatarName?.split('').join(' ')
```
Comment (jigarshah1902) [inline src/pages/PageTransferAttend/index.tsx:44]: use `let` instead of `var`

File: src/pages/PageTransferDial/index.tsx [hunk 3]
```diff
@@ -10,75 +11,86 @@ import { ScrollView, View } from 'react-native'
...
 class PageTransferDial extends React.Component {
   prevId?: string
+
   componentDidMount() {
     this.componentDidUpdate()
   }
+
   componentDidUpdate() {
-    if (this.prevId && this.prevId !== callStore.currentCall?.id) {
+    const currentCall: any = callStore.currentCall || {}
```
Comment (jigarshah1902) [inline src/pages/PageTransferDial/index.tsx:24]: Destructure object

File: src/pages/PageTransferDial/index.tsx [hunk 4]
```diff
@@ -10,75 +11,86 @@ import { ScrollView, View } from 'react-native'
...
 import contactStore from 'stores/contactStore'
 import Nav from 'stores/Nav'
-import CustomColors from 'utils/CustomColors'
 
 @observer
 class PageTransferDial extends React.Component {
   prevId?: string
+
   componentDidMount() {
     this.componentDidUpdate()
```
Comment (jigarshah1902) [inline src/pages/PageTransferDial/index.tsx:20]: What is the need here? We should not call any lifecycle method directly. React handles these calls.

**Review by jigarshah1902 — CHANGES_REQUESTED** [review]

File: src/pages/PageTransferDial/index.tsx (hunk 4 above)
Comment (Akhil-eastern) [reply src/pages/PageTransferDial/index.tsx:20]: This is existing code Jigar, I haven't written this part. I will remove it

**Review by jigarshah1902 — APPROVED** [review]

**Pull request merged**